src/debugger_requests.jl

```diff
@@ -65,8 +64,7 @@ function exec_notification(conn, state::DebuggerState, params::ExecArguments)
 
     ex = Meta.parse(params.code)
 
-    state.top_level_expressions, _ = JuliaInterpreter.split_expressions(Main, ex)
-    state.current_top_level_expression = 0
+    state.expr_splitter = JuliaInterpreter.ExprSplitter(Main, ex) # TODO: line numbers ?
```

Comment:
If you don't want the line number, the actual statement should always be `ex.args[2]`.

Reply:
I thought that we may need to fix line numbers for this code block (`params.code`) like [what we did for Juno](https://github.com/JunoLab/Atom.jl/blob/master/src/debugger/stepper.jl#L87-L93). It's actually not related to the update of JuliaInterpreter, and we can improve this later.